src/GovUk.Education.ExploreEducationStatistics.Content.Services/ThemeService.cs

```diff
@@ -110,17 +110,17 @@ private async Task<bool> HasAnyDataFiles(Release release)
         private static PublicationTreeNode BuildPublicationNode(Publication publication)
         {
             var latestRelease = publication.LatestPublishedRelease();
-            var legacyPublicationUrlIgnored = latestRelease != null;
+            var type = latestRelease == null ? PublicationType.Legacy : GetPublicationType(latestRelease.Type);
```

Comment:
Couldn't we bundle this all up into `GetPublicationType`'s switch expression:

```cs
private static PublicationType GetPublicationType(ReleaseType? releaseType)
{
   return releaseType switch
   {
     ....
     null => PublicationType.Legacy,
   };
}
```

Reply:
I wasn't sure which way to go with this and opted for doing it outside prior to calling this method but you've convinced me to do it within the `switch` now. 😄  Made the parameter optional in the latest commit.